README.md

```diff
@@ -1,18 +1,21 @@
+| --------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [EDS Analytics](https://github.com/osisoft/sample-eds-eds_analytics-dotnet)                         | This sample demonstrates how to build an application that can run analytics against streams in Edge Data Store and write results back to Edge Data Store. | [.NET](https://github.com/osisoft/sample-eds-eds_analytics-dotnet)                        | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/Edge/osisoft.sample-eds-eds_analytics-dotnet?repoName=osisoft%2Fsample-eds-eds_analytics-dotnet&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2642&repoName=osisoft%2Fsample-eds-eds_analytics-dotnet&branchName=master)                                                       |
+| [EDS Azure Deployment](https://github.com/osisoft/sample-eds-eds_azure_deployment-bash_scripts)     | This sample uses bash scripts to deploy Edge Data Store to a remote Linux edge device using Azure IoT Hub and Azure IoT Edge Modules.                     | [Bash Scripts](https://github.com/osisoft/sample-eds-eds_azure_deployment-bash_scripts)   | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/Edge/osisoft.sample-eds-eds_azure_deployment-bash_scripts?repoName=osisoft%2Fsample-eds-eds_azure_deployment-bash_scripts&branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2643&repoName=osisoft%2Fsample-eds-eds_azure_deployment-bash_scripts&branchName=master)                |
```

Comment:
Also maybe not the right place to comment this, but I would personally prefer `-bash` instead of `-bash-scripts`.

Reply:
@dcbroad3 Would you like this change for the sample repo, the language designated in the table, or both? It should be doable, but it might be best to change that in a separate pr

Reply:
I mainly mean for the sample repo name itself. It's probably OK to call them by the longer name in the table.